std_msgs/Header header
float32 input_velocity_cmd
float32 input_theta_cmd
int32 xian_dj_car_chassis_params_server_heart_beat
int32 xian_dj_car_chassis_diff_driver_control_heart_beat
int32 xian_dj_car_chassis_base_electric_left_heart_beat
int32 xian_dj_car_chassis_base_electric_right_heart_beat
int32 xian_dj_car_chassis_error_monitor_heart_beat
int32 xian_dj_car_chassis_back_end_heart_beat
float32 xian_dj_car_chassis_left_wheel_write_velocity
float32 xian_dj_car_chassis_left_wheel_read_velocity
float32 xian_dj_car_chassis_right_wheel_write_velocity
float32 xian_dj_car_chassis_right_wheel_read_velocity
float32 xian_dj_car_chassis_delta_theta1c
float32 xian_dj_car_chassis_delta_theta2c
int32 xian_dj_car_chassis_params_server_error
int32 xian_dj_car_chassis_diff_driver_control_error
int32 xian_dj_car_chassis_base_electric_error
int32 xian_dj_car_chassis_back_end_error
int32 xian_dj_car_chassis_error_code